### ISSUE TITLE
Support more <meta content=""> when translating HTML

### ISSUE THREAD

**Issue by nclm** [translate/translate#6097]
Follow up of https://github.com/translate/translate/issues/5325

Those Open Graph and Twitter tags are still missing to be translated (alt text for social images):
- `og:image:alt`
- `twitter:image:alt`

Additionally, this one should ideally be adapted to match the current locale (just like `<html lang="">`) when the `og` tags are translated:
- `og:locale`

Other less common `og` tags that could be translated include:
- `video:actor:role`
- `video:tag`
- `article:section`
- `article:tag`
- `payment:description`

Reference: https://ogp.me.
